kolibri/core/assets/src/views/language-switcher/LanguageSwitcherList.vue

```diff
@@ -51,6 +52,18 @@
   import languageSwitcherMixin from './mixin';
   import LanguageSwitcherModal from './LanguageSwitcherModal';
 
+  const prioritized_languages = ['en', 'ar', 'es-419', 'hi-in', 'fr-fr', 'sw-tz'];
+
+  const language_priorities = {};
+
+  prioritized_languages.forEach((lang, index) => {
+    language_priorities[lang] = index + 1;
+  });
+
+  const selectableLanguages = Object.values(availableLanguages).filter(
+    lang => lang.id !== currentLanguage
+  );
```

Comment:
`availableLanguages` isn't properly initialized until the component has mounted, so this doesn't currently work on refresh